.github/workflows/scripts/e2e.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
what's the numbers here? I see it's being used as 'total_time' below, but I don't follow why and where it comes from

Reply:
they are taken from `cargo test` output, an example here https://github.com/anoma/namada/actions/runs/3469076888/jobs/5795628173

Reply:
this means that we will have to update this file everytime we add a new e2e test, which I thinks its reasonable and can be automated later